app/helpers/application_helper.rb

```diff
@@ -47,8 +47,13 @@ def profiles
     profiles
   end
 
-  def transfer_statuses_for(object)
-    object.transfer_statuses.order(created_at: :desc).limit(3)
+  def set_title(title, type = nil, subtype = nil)
+    if !type.blank? && !subtype.blank?
+      title = "#{title} (#{type}) / (#{subtype})"
```

Comment:
If this could be changed to the following (and the corresponding test), the rest looks good: 

      title = "#{title} (#{type} / #{subtype})"

Not a blocker, but two separate elements in parens delimited by a / not in parens visually bugs me. :-)